spec/features/animals/upload_spec.rb

```diff
@@ -22,7 +22,7 @@
     within('tbody') { expect(page).to have_xpath('.//tr', count: 2) }
 
     # Verify no duplicates and assigned to correct org
-    animals = Animal.where(pii_tag: 2)
+    animals = Animal.where(tag: "Y2")
```

Comment:
If you take my suggestion about the factory above, you'll probably have a test to fix here.